packages/core/src/app/ui/modal/ModalTrigger.tsx

```diff
@@ -49,6 +49,8 @@ export default class ModalTrigger extends Component<ModalTriggerProps, ModalTrig
     }
 
     private handleOpen: () => void = () => {
+        document.body.scrollIntoView();
```

Comment:
`<ModalTrigger>` is being used in four components.

This bug does not apply to the normal checkout because all modals have a fixed top position due to CSS. Only embedded checkout requires a method to scroll the top of the iframe into the current view.